agent/grpc-external/services/resource/read_test.go

```diff
@@ -0,0 +1,147 @@
+package resource
+
+import (
+	context "context"
+	"testing"
+
+	"github.com/hashicorp/consul/internal/resource"
+	storage "github.com/hashicorp/consul/internal/storage"
+	"github.com/hashicorp/consul/internal/storage/inmem"
+	pbresource "github.com/hashicorp/consul/proto-public/pbresource"
+	"github.com/hashicorp/consul/proto/private/prototest"
+	"github.com/stretchr/testify/require"
+	"google.golang.org/grpc/codes"
+	"google.golang.org/grpc/metadata"
+	"google.golang.org/grpc/status"
+)
```

Comment:
Lots of folks on the team have their editor set up to split imports into stdlib, project, and 3rd party groups. It's what you get if you run `goimports -local github.com/hashicorp/consul` but I think there's an equivalent [snippet](https://stackoverflow.com/questions/38704769/golang-import-grouping-by-package) for VS Code and GoLand.

I don't think it's a hard rule, but it'll reduce some diff noise for folks that work on this file later. 😄 

```suggestion
import (
	"context"
	"testing"

	"github.com/hashicorp/consul/internal/resource"
	"github.com/hashicorp/consul/internal/storage"
	"github.com/hashicorp/consul/internal/storage/inmem"
	"github.com/hashicorp/consul/proto-public/pbresource"
	"github.com/hashicorp/consul/proto/private/prototest"
	
	"github.com/stretchr/testify/require"
	"google.golang.org/grpc/codes"
	"google.golang.org/grpc/metadata"
	"google.golang.org/grpc/status"
)
```

Reply:
I ran `goimports` as described and got
```
import (
        context "context"
        "testing"

        mock "github.com/stretchr/testify/mock"
        "github.com/stretchr/testify/require"
        "google.golang.org/grpc/codes"
        "google.golang.org/grpc/metadata"
        "google.golang.org/grpc/status"

        "github.com/hashicorp/consul/internal/resource"
        storage "github.com/hashicorp/consul/internal/storage"
        "github.com/hashicorp/consul/internal/storage/inmem"
        pbresource "github.com/hashicorp/consul/proto-public/pbresource"
        "github.com/hashicorp/consul/proto/private/prototest"
)
```

Ignoring the duplicate aliases, it put `github.com/hashicorp/consul` imports at the end instead of the middle. I don't see any additional flags on `goimports` to control this behavior.